### PR TITLE
Remove architecture from distributed tutorial

### PR DESCRIPTION
**Pull request by chester-leung**
* Reduces documentation build time
* Updates example certificates

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -22,22 +22,22 @@
...
-  filename, _ = urllib.request.urlretrieve('https://s3-us-west-2.amazonaws.com/xgboost-docs/{}.tar.bz2'.format(git_branch[0]))
-  call('if [ -d tmp ]; then rm -rf tmp; fi; mkdir -p tmp/jvm; cd tmp/jvm; tar xvf {}'.format(filename), shell=True)
-except HTTPError:
-  print('JVM doc not found. Skipping...')
-try:
-  filename, _ = urllib.request.urlretrieve('https://s3-us-west-2.amazonaws.com/xgboost-docs/doxygen/{}.tar.bz2'.format(git_branch[0]))
-  call('mkdir -p tmp/dev; cd tmp/dev; tar xvf {}; mv doc_doxygen/html/* .; rm -rf doc_doxygen'.format(filename), shell=True)
-except HTTPError:
-  print('C API doc not found. Skipping...')
+#  git_branch = os.getenv('SPHINX_GIT_BRANCH', default=None)
```
Comment (podcastinator) [inline doc/conf.py:25]: What was this doing?

File: doc/tutorials/distributed.rst [hunk 2]
```diff
@@ -109,17 +43,3 @@ If you are using Azure Confidential Computing, all your VMs must be on the same
 
       secure-xgboost/host/dmlc-core/tracker/dmlc-submit --cluster local --num-workers <num_nodes_in_cluster> --worker-memory 1g python3 distr-training.py
 
-  
-
-***************
```
Comment (podcastinator) [inline doc/tutorials/distributed.rst:114]: There is a similar troubleshooting section in the multi-client tutorial too. Can you remove that as well?

File: doc/conf.py (hunk 1 above)
Comment (chester-leung) [reply doc/conf.py:25]: I believe it was making calls to S3 to fetch the JVM and the C API docs, which we don't need.

File: doc/tutorials/distributed.rst (hunk 2 above)
Comment (chester-leung) [reply doc/tutorials/distributed.rst:114]: Done

**Comment by podcastinator** [comment]
Thanks!

**Review by podcastinator — APPROVED** [review]

**Pull request merged**